src/Components/SmartComponents/Modals/OsExposureReportModal/LifecycleSelect.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import {
+    Select,
+    MenuToggle,
+    SelectList,
+    SelectOption
+} from '@patternfly/react-core';
+
+const LifecycleSelect = ({
+    minorVersion,
+    minorVersionDetails
+}) => {
+    const [isSelectOpen, setIsSelectOpen] = useState(false);
+    const [selectedValue, setValue] = useState('Minor release');
+
+    return (
+        <Select
+            id="select-id"
```

Comment:
Here since theres multiple, to help out react I think itd be good to use here
https://react.dev/reference/react/useId

const id = useId() and slap that in the id field. This is more important when the array is dynamic